host/allwinner/gpio.go

```diff
@@ -165,16 +165,19 @@ func (p *Pin) In(pull gpio.Pull, edge gpio.Edge) error {
 	return nil
 }
 
-// Read returns the current level of the pin. Due to the way the Allwinner hardware functions it
-// will do this regardless of the pin's function but this should not be relied upon.
+// Read returns the current level of the pin.
+//
+// Due to the way the Allwinner hardware functions it will do this regardless
```

Comment:
what do you mean by `it will do this`?

Reply:
I'd have to ask @tve , I forget the rationale. I'll remove the comment as I don't understand what it means either. :)